controller/backup_target_controller.go

```diff
@@ -540,6 +543,14 @@ func (btc *BackupTargetController) getInfoFromBackupStore(backupTarget *longhorn
 	defer engineClientProxy.Close()
 
 	// Get required information using backup target client.
+	// Get SystemBackups first to update the backup target to `available` while minimizing requests to S3.
+	info.backupStoreSystemBackups, err = backupTargetClient.ListSystemBackup()
+	if err != nil {
+		return backupStoreInfo{}, errors.Wrapf(err, "failed to list system backups in %v", backupTargetClient.URL)
+	}
+	if !backupTarget.Status.Available {
+		return info, nil
+	}
```

Comment:
Do we have other kinds of methods to verify if the backup target is available/reachable or not?

Reply:
No. IIUC, the only way to check if the backup target is available or reachable from the' longhorn-manager' is to execute the engine binary command.

Reply:
OK. NTW, the comment is a little bit confusing as well. How about:
```
If the controller can communicate with the remote backup target while "backupTarget.Status.Available" is "false", Longhorn should update the field to "true" first rather than continuing to fetch info from the target.
```

Reply:
Modified the comment. Thanks.